components/button/Button.module.css

```diff
@@ -1,8 +1,12 @@
 .btn {
-  border-radius: 10px;
+  border-radius: 35px;
+  border: none;
+  color: var(--primary-font-color);
+  background: var(--secondary-color);
+  padding: 0.8rem;
 }
 
 .primary {
-  color: antiquewhite;
-  background: black;
+  color: var(--primary-font-color);
+  background: var(--secondary-color);
 }
```

Comment:
It doesn't matter if you add `.primary` or not, because `color` and `background` are similar to the base class `.btn`.
Maybe you can remove the `primary` property.